### PR TITLE
Document patch release 4.0.15 RNs

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR adds enhancements and bug fixes for the 4.0.15 patch release.

## Review Changes

💻 [Preview](https://deploy-preview-1633--docs-spectrocloud.netlify.app/release-notes)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-882)

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
@ritawatson this does not need the 3.4 backport tag as version 4.0 content don't belong in that branch.

File: docs/docs-content/release-notes.md [hunk 1]
```diff
@@ -10,7 +10,27 @@ sidebar_custom_props:
...
+
+- Configurable options to specify charging for Azure Kubernetes Service (AKS) cluster control planes are now available in the Kubernetes `values.yaml` pack. These options are based on Service Level Agreements (SLA) that Azure provides. The ability to set these options in the Kubernetes YAML file allows you to embed options and overrides in cluster profiles.
+
+  Use the `managedControlPlane.sku` parameter to specify Azure recommended settings: `Standard` for production clusters and `Free` for non-production or small clusters.
+
+- Palette now supports the space character for the `scopesDelimiter` field for OpenID Connect (OIDC) configuration. Previously, Palette used only a comma delimiter.
+
+### Bug Fixes
+
+- API calls to update edge-native machines no longer time out after 60 seconds.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/release-notes.md:28]:
```suggestion
- API calls to update edge hosts no longer time out after 60 seconds. 
```
We don't use the edge native term as it's only applicable to GE.

File: docs/docs-content/release-notes.md [hunk 2]
```diff
@@ -10,7 +10,27 @@ sidebar_custom_props:
...
+
+## October 3, 2023 - Release 4.0.15
+
+This release contains enhancements and minor bug fixes.
+
+## Enhancements
+
+- Configurable options to specify charging for Azure Kubernetes Service (AKS) cluster control planes are now available in the Kubernetes `values.yaml` pack. These options are based on Service Level Agreements (SLA) that Azure provides. The ability to set these options in the Kubernetes YAML file allows you to embed options and overrides in cluster profiles.
+
+  Use the `managedControlPlane.sku` parameter to specify Azure recommended settings: `Standard` for production clusters and `Free` for non-production or small clusters.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/release-notes.md:22]: @ritawatson it feels like we are putting too much information here. Can we instead add a small heading in the Azure architecture page, document the feature, and link to it here? The Jira ticket has the YAML values, which would be good to capture.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Minor suggestion

**Review by velguruswami — COMMENTED** [review]
For PE-2595 - here is description of the fix - "The issue with showing incorrect interface of an edge host during cluster deployment is now resolved."

**Review by karl-cardenas-coding — APPROVED** [review]
Looks great 😄

**Comment by karl-cardenas-coding** [comment]
@ritawatson I fixed the issue and made sure the changes got push to the `version-4-0` branch 😄

**Pull request merged**